ethereum-consensus/src/capella/block_processing.rs

```diff
@@ -37,10 +41,33 @@ pub fn process_bls_to_execution_change<
         BYTES_PER_LOGS_BLOOM,
         MAX_EXTRA_DATA_BYTES,
     >,
-    _signed_address_change: &mut SignedBlsToExecutionChange,
-    _context: &Context,
+    signed_address_change: &mut SignedBlsToExecutionChange,
+    context: &Context,
 ) -> Result<()> {
-    unimplemented!()
+    let address_change = signed_address_change;
+
+    assert!(address_change.message.validator_index < state.validators.len());
+
+    let validator = &mut state.validators[address_change.message.validator_index];
+
+    assert!(validator.withdrawal_credentials.starts_with(&[BLS_WITHDRAWAL_PREFIX]));
```

Comment:
this works but I don't see a reason to not just stick more closely to the spec
```suggestion
    assert!(validator.withdrawal_credentials[0] == BLS_WITHDRAWAL_PREFIX);
```

we know the direct element access is safe as the type of `withdrawal_credentials` is always at least that big

Reply:
done